ui/components/common/GraphQLFormatters.js

```diff
@@ -0,0 +1,20 @@
+const formatType = type => {
```

Comment:
Not a component, should be located out of this folder. (same goes for validators but.. too late)